packages/mobile/src/escrow/saga.ts

```diff
@@ -117,6 +217,125 @@ function* registerStandbyTransaction(context: TransactionContext, value: string,
   )
 }
 
+async function formEscrowWithdrawTxWithNoCode(
+  contractKit: ContractKit,
+  escrowWrapper: EscrowWrapper,
+  paymentId: string,
+  pepper: string,
+  msgHash: string,
+  addressIndex: number
+) {
+  const { privateKey } = generateDeterministicInviteCode(pepper, addressIndex)
+  const signature: string = (await contractKit.web3.eth.accounts.sign(msgHash, privateKey))
+    .signature
+  Logger.debug(
+    TAG + '@withdrawFromEscrowViaKomenci',
+    `Signed message hash signature is ${signature}`
+  )
+
+  const { r, s, v } = splitSignature(contractKit, signature)
+  const withdrawTx = escrowWrapper.withdraw(paymentId, v, r, s)
+  return withdrawTx
+}
+
+function* withdrawFromEscrowUsingPepper(komenciActive: boolean = false) {
+  try {
+    ValoraAnalytics.track(OnboardingEvents.escrow_redeem_start)
+    Logger.debug(TAG + '@withdrawFromEscrowViaKomenci', 'Withdrawing escrowed payment')
```

Comment:
Should the tag in this logging statement match the function name?
```suggestion
    Logger.debug(TAG + '@ withdrawFromEscrowUsingPepper', 'Withdrawing escrowed payment')
```